Cmd arrow

henri123lemoine opened the pull request:
resolve #23

budzianowski commented:
It would be awesome to port this to https://github.com/kscalelabs/sim

jingxiangmo commented:
> It would be awesome to port this to https://github.com/kscalelabs/sim

I opened a PR already

jingxiangmo commented:
@henri123lemoine could you please add a argument flag like `--command-arrow` so it's not turned on every time?

henri123lemoine commented:
> @henri123lemoine could you please add a argument flag like `--command-arrow` so it's not turned on every time?

Done.

Also cleaned up a few other things, and added a command manager class, mainly for modularity and to make future improvements easier.
